engine/src/physics/plugin.cpp

```diff
@@ -81,16 +143,65 @@ void cubos::engine::physicsPlugin(Cubos& cubos)
                 auto force = Force{};
                 force.add(bundle.force);
 
+                auto torque = Torque{};
+                torque.add(bundle.torque);
+
                 auto impulse = Impulse{};
                 impulse.add(bundle.impulse);
 
+                auto angularImpulse = AngularImpulse{};
+                angularImpulse.add(bundle.angularImpulse);
+
                 cmds.add(ent, Mass{.mass = bundle.mass, .inverseMass = 1.0F / bundle.mass});
+                if (bundle.inertiaTensor != glm::mat3(0.0F))
+                {
+                    cmds.add(ent, Inertia{.inertia = bundle.inertiaTensor,
+                                          .inverseInertia = glm::inverse(bundle.inertiaTensor),
+                                          .autoUpdate = false});
+                }
                 cmds.add(ent, Velocity{.vec = bundle.velocity});
+                cmds.add(ent, AngularVelocity{.vec = bundle.angularVelocity});
                 cmds.add(ent, force);
+                cmds.add(ent, torque);
                 cmds.add(ent, impulse);
+                cmds.add(ent, angularImpulse);
                 cmds.add(ent, AccumulatedCorrection{});
             }
         });
 
+    // Should this be here?
+    cubos.system("update inertia - box collider")
+        .tagged(physicsPrepareSolveTag)
+        .call([](Query<Mass&, BoxCollisionShape&, Inertia&> query) {
+            for (auto [mass, shape, inertia] : query)
+            {
+                if (!inertia.autoUpdate)
+                {
+                    continue;
+                }
+
+                if (!mass.changed && !shape.changed)
+                {
+                    continue;
+                }
+
+                mass.changed = false;
+                shape.changed = false;
+
+                // Object has infinite mass
+                if (mass.inverseMass == 0.0F)
+                {
+                    inertia.inertia = glm::mat3(0.0F);
+                    inertia.inverseInertia = glm::mat3(0.0F);
```

Comment:
Once again, is this supposed to be an identity matrix or a zero matrix?

Reply:
Here it's another case, the inverseInertia is correct, the inertia itself I'm still not sure what to put bc it should be infinite, which is the same issue we currently have with mass. This doesn't affect anything for now bc we don't use it directly, but we should have a better way to identify these edge cases probably. I still haven't looked into it bc I'm pretty sure that when we introduce proper static bodies and such, this will naturally come into discussion again, but I'll create an issue so I don't forget.